bentoml/deployment/sagemaker/templates.py

```diff
@@ -86,7 +86,9 @@
 from bentoml.server.utils import get_gunicorn_num_of_workers
 
 bento_server_timeout = os.environ.get('BENTO_SERVER_TIMEOUT', 60)
-bento_server_workers = int(os.environ.get('BENTO_SERVER_WORKERS', get_gunicorn_num_of_workers()))
+bento_server_workers = int(
```

Comment:
let's move this to a separate python file? so we got to run linter and unit test ?

Reply:
actually same for the DEFAULT_WSGI_PY and DEFAULT_NGINX_CONFIG above